README.md

```diff
@@ -240,6 +240,54 @@ logging:
 | | `.level.io.camunda.migrator` | `string`  | Migrator logging level. Default: `INFO`                                                                                                   |
 | | `.file.name`                 | `string`  | Log file location. Set to: `logs/c7-data-migrator.log`. If not specified, logs are output to the console.                                 |
 
+## Variable transformation
```

Comment:
/suggest I would suggest describing high-level how to build an extension JAR and where to drop it.

Reply:
Will follow up in https://github.com/camunda/c7-data-migrator/pull/113